app/prefix_test.go

```diff
@@ -0,0 +1,59 @@
+package app
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFullBIP44PathForTestnet(t *testing.T) {
+	cases := []struct {
+		name         string
+		expected     string
+		isTestnet    bool
+		seal         bool
+		shouldPanic  bool
+		panicMessage string
+	}{
+		{
+			"has correct bip44th path for testnet",
+			"m/44'/118'/0'/0/0",
+			true,
+			false,
+			false,
+			"",
+		},
+		{
+			"has correct bip44th path for mainnet",
+			"m/44'/118'/0'/0/0",
+			true,
```

Comment:
Good catch.

Looks like these tests weren't actually doing anything. `118` is the `atom` coin type. We want it to be `505` for mainnet and `1` for testnet. The test was erroneously just getting the BIP44 of the default config each time rather than the config being set by our `SetConfig`.

I've corrected these tests.